release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -409,6 +409,13 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-release-notes-insights-operator-enhancements_{context}"]
 === Insights Operator
 
+==== Insights Runtime Extractor (Technology Preview)
+
+In {product-title} 4.18, the Insights Operator introduces a new workload data collection feature, _Insights Runtime Extractor_ (`InsightsRuntimeExtractor`), to help Red{nbsp}Hat better understand the workload of your containers.
```

Comment:
Not sure what type of thing the `InsightsRuntimeExtractor` in backticks is - a CRD? Does that actually need mention in a RN blurb at all, or does it make sense to just use the "Insights Runtime Extractor" feature name here and throughout the RN text?
```suggestion
In this release, the Insights Operator introduces the workload data collection _Insights Runtime Extractor_ feature to help Red{nbsp}Hat better understand the workload of your containers.
```

Reply:
It's the actual name of the feature resource that customers enable through feature gates. 
We also add and refer this feature to the 'Enabling feature gates' topic too, and use this same format, similar to other data collection features/resources that are in Tech preview.